e2e-tests/insurance/cypress/e2e/journeys/your-buyer/check-your-answers/check-your-answers-summary-list/check-your-answers-summary-list.spec.js

```diff
@@ -81,13 +86,36 @@ context('Insurance - Your buyer - Check your answers - Summary list - your buyer
       checkSummaryList[CONNECTION_WITH_BUYER_DESCRIPTION]({ shouldRender: false });
     });
 
-    // TODO: EMS-2628
-    // it(`should render a ${TRADED_WITH_BUYER} summary list row`, () => {
-    //   checkSummaryList[TRADED_WITH_BUYER]();
-    // });
+    it(`should render a ${TRADED_WITH_BUYER} summary list row`, () => {
+      checkSummaryList[TRADED_WITH_BUYER]();
+    });
+
+    it(`should render a ${TRADED_WITH_BUYER} summary list row`, () => {
+      checkSummaryList[TRADED_WITH_BUYER]();
+    });
+
+    it(`should render a ${TRADED_WITH_BUYER} summary list row`, () => {
+      checkSummaryList[TRADED_WITH_BUYER]();
+    });
+
+    it(`should render a ${OUTSTANDING_PAYMENTS} summary list row`, () => {
+      checkSummaryList[OUTSTANDING_PAYMENTS]();
+    });
+
+    it(`should not render a ${TOTAL_AMOUNT_OVERDUE} summary list row`, () => {
+      checkSummaryList[TOTAL_AMOUNT_OVERDUE]({ shouldRender: false });
+    });
+
+    it(`should not render a ${TOTAL_OUTSTANDING_PAYMENTS} summary list row`, () => {
+      checkSummaryList[TOTAL_OUTSTANDING_PAYMENTS]({ shouldRender: false });
+    });
+
+    it(`should render a ${FAILED_PAYMENTS} summary list row`, () => {
+      checkSummaryList[FAILED_PAYMENTS]();
+    });
```

Comment:
Not a blocker, tech improvement could be made in the future, could move these summary list assertions into `shared-test-assertions`